bokeh/client/session.py

```diff
@@ -164,10 +165,7 @@ def show_session(session_id=None, url='default', app_path='/',
             server_url = coords.server_url
             session_id = coords.session_id
 
-        if controller is None:
-            from bokeh.util.browser import get_browser_controller
```

Comment:
Sure, I did not see your comment ... Sorry for the rebased PR, promise it will not happen again.